File: lib_customview/src/main/java/com/caesarlib/customview/AwesomeFontTextView.java
package com.caesarlib.customview;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatTextView;

import com.caesarlib.res_tools.FontTool;

public class AwesomeFontTextView extends AppCompatTextView {


    public AwesomeFontTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
        setTypeface(FontTool.INSTANCE.getTypeface(context));
    }
}

File: lib_customview/src/main/java/com/caesarlib/customview/AwesomeFontTextView.kt
package com.caesarlib.customview

import android.content.Context
import android.util.AttributeSet

import androidx.appcompat.widget.AppCompatTextView

import com.caesarlib.res_tools.FontTool

class AwesomeFontTextView(context: Context, attrs: AttributeSet) : AppCompatTextView(context, attrs) {


    init {
        typeface = FontTool.getTypeface(context)
    }
}
